pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -161,6 +154,10 @@ func defaultStrategyChooser(instance *v1.Jaeger, secrets []corev1.Secret) strate
 }
 
 func (r *ReconcileJaeger) apply(jaeger v1.Jaeger, str strategy.S) error {
+	if err := r.applyUpgrades(jaeger); err != nil {
+		return err
+	}
```

Comment:
Should the operator continue if the version of the instance does not match a version which can be managed by the operator?

Reply:
This PR removes the ability for the CR to specify its own Jaeger version number. It can certainly still override the image, but whenever a CR is created, we just assume that it's the same version of the operator that is running (Operator v1.13.1 = Jaeger v1.13.1).

Reply:
I guess I missed one part: this all means that there's no version that can't be managed by the operator :-)